core/vm/contracts.go

```diff
@@ -202,30 +203,21 @@ func init() {
 	addrsList = append(addrsList, PrecompiledAddressesApricotPhasePre6...)
 	addrsList = append(addrsList, PrecompiledAddressesApricotPhase6...)
 	addrsList = append(addrsList, PrecompiledAddressesBanff...)
+	addrsList = append(addrsList, genesisContractAddr, NativeAssetBalanceAddr, NativeAssetCallAddr) // Add genesis multicoin contract and deprecated native asset precompiles to set of reserved addresses
```

Comment:
why is this needed? seems they are included in PrecompiledContractsApricotPhase2

Reply:
Good catch